client/my-sites/backup/main.jsx

```diff
@@ -214,11 +206,7 @@ function BackupStatus( { selectedDate, needCredentials, onDateChange } ) {
 							</div>
 						</div>
 						<div className="backup__header-right">
-							{ siteSlug && (
-								<>
-									<BackupActionsToolbar siteId={ siteId } siteSlug={ siteSlug } />
-								</>
-							) }
+							<BackupActionsToolbar siteId={ siteId } />
```

Comment:
`siteSlug` validation is no longer needed, as we refactored the component to query the siteSlug directly.